src/TVideoPlayer.cpp

```diff
@@ -0,0 +1,24 @@
+/***************************************************************************
```

Comment:
This file is currently redundant  - the trivial constructor can be placed within the header file (`src/TVideoPlayer.h`) as is the case with some other classes...